Formula/h/halide.rb

```diff
@@ -30,39 +29,24 @@ class Halide < Formula
   depends_on "lld"
   depends_on "llvm"
   depends_on "python@3.13"
-
-  # Check wabt version in `dependencies/wasm/CMakeLists.txt`.
-  # TODO: Ask upstream to support usage of a system-provided wabt.
-  # TODO: Do we really need a git checkout here?
-  resource "wabt" do
-    url "https://github.com/WebAssembly/wabt.git",
-        tag:      "1.0.33",
-        revision: "963f973469b45969ce198e0c86d3af316790a780"
-  end
+  depends_on "wabt"
 
   def python3
     "python3.13"
   end
 
   def install
-    builddir = buildpath/"build"
-    (builddir/"_deps/wabt-src").install resource("wabt")
-
     site_packages = prefix/Language::Python.site_packages(python3)
     rpaths = [rpath, rpath(source: site_packages/"halide")]
     args = [
       "-DCMAKE_INSTALL_RPATH=#{rpaths.join(";")}",
       "-DHalide_INSTALL_PYTHONDIR=#{site_packages}",
```

Comment:
```suggestion
      "-DHalide_INSTALL_PYTHONDIR=#{site_packages}/halide",
```

We were forced to remove the automatic `halide` suffix to correctly configure scikit-build-core for the Pip package.

Reply:
Worse, we couldn't work around it by setting `Halide_INSTALL_PYTHONDIR` to `..`. This would resolve to `../halide` and then we ran into a bug in CMake that I actually filed with Kitware. See here: https://gitlab.kitware.com/cmake/cmake/-/issues/26252

It was fixed by this policy in CMake 3.31: https://cmake.org/cmake/help/latest/policy/CMP0177.html